apps/src/lib/node/ledger/shell/governance.rs

```diff
@@ -136,6 +136,20 @@ where
                             id
                         );
 
+                        for ibc_event in
```

Comment:
It looks to me like we are doing this loop to set the height for ibc events three times: once here, once in the `apply_inflation` function and once when we handle the `TxResult` coming from `dispatch_tx`. I wonder if we could do this just once at the end of `finalize_block`

Reply:
Nevermind, this doesn't make sense, we need to extract the events from the write log and push them to the abci response